ci/scripts/make_apidocs.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+
+# In this script, we:
```

Comment:
this is the main thing in this PR: the rest is autogenerated files or files where I had to update the license links